.vscode/settings.json

```diff
@@ -0,0 +1,9 @@
+// Place your settings in this file to overwrite default and user settings.
```

Comment:
Add `.vscode` to the `.gitignore` file and remove this from the PR. LMK if I can give you a hand w/ the `git` commands for all that.